src/main/generic/consensus/primitive/Primitive.js

```diff
@@ -14,7 +14,7 @@ class Primitive {
     }
 
     toString() {
-        return this.toBase64();
+        return this.toHex().toUpperCase();
```

Comment:
Yes, I was thinking about this, too. I now think that changing it at the actual displaying code is better than at the root. That way no changing/breaking behaviour is introduced and the control is still in the display code.

I will change this later today.